ckanext/s3filestore/uploader.py

```diff
@@ -409,17 +416,19 @@ def __init__(self, resource):
             resource['url'] = self.filename
             resource['url_type'] = 'upload'
             resource['last_modified'] = datetime.datetime.utcnow()
+
+            self.upload_file = _get_underlying_file(upload_field_storage)
```

Comment:
what is this doing?

Reply:
Just to match Keitaro. It should be functionally the same.